client/cypress/integration/event-manager/signup_spec.js

```diff
@@ -1,7 +1,9 @@
+import { randomData } from './dataSet';
 describe('Sign up', function() {
   context('Testing Signup Page', function() {
     it('Load signup page', function() {
       cy.visit('/signup');
+      cy.get('div').should('have.class', 'login-container');
```

Comment:
@taleymirza how do we know it hasnt opened the login view? Assertions need to be specific